app/src/main/kotlin/net/syncthing/lite/dialogs/downloadfile/DownloadFileStatus.kt

```diff
@@ -0,0 +1,12 @@
+package net.syncthing.lite.dialogs.downloadfile
+
+import java.io.File
+
+sealed class DownloadFileStatus
+object DownloadFileStatusFailed: DownloadFileStatus()
+data class DownloadFileStatusDone(val file: File): DownloadFileStatus()
+data class DownloadFileStatusRunning(val progress: Int): DownloadFileStatus() {
+    companion object {
+        const val MAX_PROGRESS = 1000
```

Comment:
Any reason you are using 1000 instead of 100 (percentage), or the file size here?

Reply:
It allows seeing the progress of bigger files better (because there is a shorter time between the steps). However, I don't think anyone would (or could, as the received files are stored in the RAM until the download is done, but that's another story) use this app for large files.